src/Aspire.Dashboard/Components/Pages/TraceDetail.razor

```diff
@@ -181,6 +181,6 @@
 else
 {
     <div class="empty-content">
-        <FluentIcon Icon="Icons.Regular.Size24.GanttChart" /> &nbsp; @Loc[nameof(Dashboard.Resources.TraceDetail.TraceDetailTraceNotFound)]
+        <FluentIcon Icon="Icons.Regular.Size24.GanttChart" /> &nbsp; @string.Format(Loc[nameof(Dashboard.Resources.TraceDetail.TraceDetailTraceNotFound)], TraceId)
```

Comment:
Nothing complains about passing the right culture info?

Reply:
Not in razor files. Here the argument `TraceId` is a string, so there is no culture conversion involved.